[2023-05-23T23:13:49.734+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_todays_gamesv3.postgres_to_s3_task manual__2023-05-23T23:13:33.775763+00:00 [queued]>
[2023-05-23T23:13:49.748+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_todays_gamesv3.postgres_to_s3_task manual__2023-05-23T23:13:33.775763+00:00 [queued]>
[2023-05-23T23:13:49.748+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T23:13:49.748+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-05-23T23:13:49.748+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-05-23T23:13:49.766+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): postgres_to_s3_task> on 2023-05-23 23:13:33.775763+00:00
[2023-05-23T23:13:49.772+0000] {standard_task_runner.py:55} INFO - Started process 189 to run task
[2023-05-23T23:13:49.780+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_todays_gamesv3', 'postgres_to_s3_task', 'manual__2023-05-23T23:13:33.775763+00:00', '--job-id', '460', '--raw', '--subdir', 'DAGS_FOLDER/todays_gamesv3.py', '--cfg-path', '/tmp/tmps0kw70xz']
[2023-05-23T23:13:49.780+0000] {standard_task_runner.py:83} INFO - Job 460: Subtask postgres_to_s3_task
[2023-05-23T23:13:49.855+0000] {task_command.py:388} INFO - Running <TaskInstance: get_todays_gamesv3.postgres_to_s3_task manual__2023-05-23T23:13:33.775763+00:00 [running]> on host b03c83f459ca
[2023-05-23T23:13:49.959+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_todays_gamesv3
AIRFLOW_CTX_TASK_ID=postgres_to_s3_task
AIRFLOW_CTX_EXECUTION_DATE=2023-05-23T23:13:33.775763+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-23T23:13:33.775763+00:00
[2023-05-23T23:13:49.974+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-05-23T23:13:49.982+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/functions/postgres_to_s3.py", line 10, in postgres_to_s3
    with NamedTemporaryFile(mode='w') as f: #puts file in temp folder
NameError: name 'NamedTemporaryFile' is not defined
[2023-05-23T23:13:49.993+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=get_todays_gamesv3, task_id=postgres_to_s3_task, execution_date=20230523T231333, start_date=20230523T231349, end_date=20230523T231349
[2023-05-23T23:13:50.008+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 460 for task postgres_to_s3_task (name 'NamedTemporaryFile' is not defined; 189)
[2023-05-23T23:13:50.030+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-05-23T23:13:50.070+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
